checker/jdk/index/src/java/util/Collections.java

```diff
@@ -3349,8 +3349,9 @@ public void remove() {
      * @return an immutable list containing only the specified object.
      * @since 1.3
      */
+    @SuppressWarnings("return.type.incompatible") // #1: returns an immutable list containing only the specified object o, hence @MinLen(1)
```

Comment:
The fact that the list is immutable is irrelevant.  Therefore, please omit it, lest you mislead readers.  Readers will think that everything you write is relevant.

Also, rather than suppressing a warning, wouldn't it be better to annotate SingletonList?

Reply:
I suggested that you annotate the SingletonList class.  Then you would not have to suppress a warning here.  You didn't respond to that comment.
Could you please try it, or report why it didn't work?

Reply:
You still have not made this change.  Please make the change.